lib/internal/inspector_network_tracking.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const {
+  DateNow,
+} = primordials;
+
+let dc;
+let NodeNetwork;
+
+let requestId = 0;
+const getNextRequestId = () => `node-network-event-${++requestId}`;
+
+function onClientRequestStart({ request }) {
+  const url = `${request.protocol}//${request.host}${request.path}`;
+  const wallTime = DateNow();
+  const timestamp = wallTime / 1000;
+  request._inspectorRequestId = getNextRequestId();
+  NodeNetwork.requestWillBeSent({
```

Comment:
I tried it out today when I had free time. Is it possible to retrieve the original call location from the stack obtained in this way, which indicates `diagnostics_channel`?

Reply:
Does 'the original call location' refer to the location in the user script where the network activity started? I mean, are you looking to retrieve the location indicated below?
```js
const res = await fetch("https://nodejs.org/en"); // <-- here
```
If so, I think it's difficult with the current `diagnostics_channel` API. cc @nodejs/diagnostics

Reply:
Yes, I mean it's not easy to do with dc.